app/views/site/_reviewer_access_fields.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="well well-sm" id="reviewer-access-controls">
+  <div class="form-group row">
+    <div class="col-md-2">
+      <% access_enabled = f.object.persisted? %>
+      <%= label_tag :_destroy, 'Access Enabled?' %><br />
+      <div class="btn-group btn-toggle" data-toggle="buttons">
+        <label class="btn btn-default <%= access_enabled ? 'active' : nil %>">
+          <%= radio_button_tag 'reviewer_opts[enable]', :yes, access_enabled, disabled: !access_enabled %> Yes
```

Comment:
see previous comment about renaming this "reviewer_access_actions" or somesuch

Reply:
446268b